packages/manager/modules/pci/src/projects/project/instances/instance/stop/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,8 @@
+{
+  "pci_projects_project_instances_instance_stop_title": "Arrêt de votre instance",
+  "pci_projects_project_instances_instance_stop_submit_label": "Confirmer",
+  "pci_projects_project_instances_instance_stop_cancel_label": "Annuler",
+  "pci_projects_project_instances_instance_stop_content": "Vous allez arrêter votre instance {{ instance }}. Les ressources dédiées à votre instance Public Cloud sont toujours réservées (adresse IP incluse). Vous pouvez redémarrer votre instance à tout moment. Dans l'intervalle, vous êtes toujours facturé au même prix pour votre instance.",
+  "pci_projects_project_instances_instance_stop_success_message": "L'arrêt de l'instance {{ instance }} effectué.",
```

Comment:
Some missing words ?

Reply:
Right i do that by myself i am waiting reporter response for new message.
The initial message is not right one for the current case.

Reply:
i got the new trad text.
so fixed.